File: Java/DailyFlash/August/19Aug/Program2.java

import java.util.Scanner;

class Weekday {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
	
		int num;

		System.out.print("Enter a number between 0 to 6: ");
		num=sc.nextInt();

		switch(num) {

			case 0:
				System.out.println("Monday");
				break;
			case 1:
				System.out.println("Tuesday");
				break;
			case 2:
				System.out.println("Wednesday");
				break;
			case 3:
				System.out.println("Thursday");
				break;
			case 4:
				System.out.println("Friday");
				break;
			case 5:
				System.out.println("Saturday");
				break;
			case 6:
				System.out.println("Sunday");
				break;
			default:
				System.out.println("U enetered number other than 0 to 6!");
				break;
		}
	}
}

File: PPA/DailyFlash/August/19August/Program2.c


#include<stdio.h>

void main() {

	int num;

	printf("Enter a number between 0 to 6: ");
	scanf("%d",&num);

	switch(num) {

		case 0:
			printf("Monday\n");
			break;
		case 1:
			printf("Tuesday\n");
			break;
		case 2:
			printf("Wednesday\n");
			break;
		case 3:
			printf("Thursday\n");
			break;
		case 4:
			printf("Friday\n");
			break;
		case 5:
			printf("Satday\n");
			break;
		case 6:
			printf("Sunday\n");
			break;
		default:
			printf("U enetered number other than 0 to 6!\n");
			break;
	}
}
